### DISCUSSION TITLE
Can't creat a NUC disk

### DISCUSSION THREAD

**Discussion by DJe73260** [home-assistant/operating-system#3664] [Q&A]
Hy
I have a NUC x86-X64 with a HA install.
I would replace the 500Go HDD by a 128 Go SSD.
I have downloaded the images 13.2 and 13.1 but i have everytime the same error from balena etcher.
"https://github.com/home-assistant/operating-system/releases/download/13.2/haos_generic-x86-64-13.2.img.xz"
![Balena](https://github.com/user-attachments/assets/e7ff41ed-823d-431d-9a7d-22f84c696aa2)

Yet here is what becomes of my disk
![Partition](https://github.com/user-attachments/assets/44d64edf-bcf0-44c3-8b55-19c000ea6b89)

does anyone have an idea ?

Thanks

**Comment by Impact123** [comment]
This is hardly a bug of the OS but try to extract it first.

**Reply by DJe73260** [reply]
Exactly !!!!!!!!!!!
1 second and it's ok
Thanks
